pkgs/tools/system/htop/default.nix

```diff
@@ -1,7 +1,14 @@
 { lib, fetchFromGitHub, stdenv, autoreconfHook
-, ncurses, IOKit
+, ncurses
+, IOKit
+, sensorsSupport ? stdenv.isLinux, lm_sensors
+, systemdSupport ? stdenv.isLinux, systemd
 }:
 
+with lib;
+
```

Comment:
lib. is so nice and cute, we don't need to use a with over the entire file.

Reply:
correct, but off-topic in this pr